packages/schematics/angular/utility/json-utils.ts

```diff
@@ -31,14 +37,67 @@ export function appendPropertyInAstObject(
   );
 }
 
+export function insertPropertyInAstObjectInOrder(
+  recorder: UpdateRecorder,
+  node: JsonAstObject,
+  propertyName: string,
+  value: JsonValue,
+  indent = 4,
```

Comment:
Default for most JSON is 2.

Reply:
Indent is defaulted to 4, because the typical use case is to add a value to an existing object (i.e. `dependencies`, `devDependencies`) It could be removed as a default to make it more explicit.